Makefile

```diff
@@ -113,7 +113,7 @@ endif
 # available only for linux host because CGO usage
 ifeq ($(HOST_OS),linux)
 docker-local-build: go-spacemesh hare p2p harness
-	cd build; docker build -f ../Dockerfile.prebuiltBinary -t $(DOCKER_IMAGE) .
+	cd build; DOCKER_BUILDKIT=1 docker build -f ../Dockerfile.prebuiltBinary -t $(DOCKER_IMAGE) .
```

Comment:
I think this is not required, `DOCKER_BUILDKIT=1` should be the default on any docker released/installed since Oct 2020.

Reply:
I'm not so sure. I'm kinda semi-sure that it's still behind a flag. Its not anymore experimental flag though.

At least that's what docs say :)

Reply:
It complained on my machine without it, so I guess it's not a default.

Reply:
I was going by this blog post: https://pythonspeed.com/articles/docker-buildkit/#:~:text=Enabling%20BuildKit%20in,Docker%20Engine.

Also on my machine with and without the Environment variable I notice little to no difference 🤷

Reply:
This blog post says it's ON by default for Mac and Windows and OFF for Linux. 

Anyway, IMHO setting it doesn't hurt at all.

Reply:
It's actually even worse. It's set only when using docker desktop app :)

And you can set docker in milion other ways that using docker desktop.

I think setting it is a must have (if it works) :).